capture/src/sinks/print.rs

```diff
@@ -10,7 +10,7 @@ pub struct PrintSink {}
 #[async_trait]
 impl Event for PrintSink {
     async fn send(&self, event: ProcessedEvent) -> Result<(), CaptureError> {
-        info!("single event: {:?}", event);
+        info!("single {:?} event: {:?}", event.data_type, event);
```

Comment:
Maybe a bit redundant since the debug print `{:?}` will already include all fields in the struct? So, `event.data_type` will already be printed.

I would maybe consider implementing a proper `std::fmt::Display` for `ProcessedEvent` and handle how we want to display it there instead of here in the sink (as with every change to the event we'll have to come back and look at the sink too).

Not a big deal nor a blocker though.

Reply:
Agreed, thanks! That's a remnant of when I passed `DataType` as a function argument instead of a struct field. Will remove